.github/workflows/pd-tests.yaml

```diff
@@ -23,9 +23,9 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        worker_id: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
```

Comment:
Do we need to modify it? To do so, you need to change the github Settings related to the required CI

Reply:
- I found that tools is the most time-consuming task, so I am checking the reason now.
- after putting tools tests in a separate chunk, the other unit test time is just 1-3 min when there are 10 chunks, so there are many redundant chunks...

> you need to change the github Settings related to the required CI

I am not sure about that, maybe you are right, we can rethink it in another pr